install/kubernetes/cilium/templates/cilium-preflight/deployment.yaml

```diff
@@ -46,20 +46,20 @@ spec:
             cilium-dbg preflight validate-cnp;
             touch /tmp/ready-validate-cnp;
             sleep 1h;
-          livenessProbe:
-            exec:
-              command:
-              - cat
-              - /tmp/ready-validate-cnp
-            initialDelaySeconds: 5
-            periodSeconds: 5
           readinessProbe:
             exec:
               command:
               - cat
               - /tmp/ready-validate-cnp
-            initialDelaySeconds: 5
-            periodSeconds: 5
+            {{- if semverCompare "<1.20-0" .Capabilities.KubeVersion.Version }}
```

Comment:
Is `1.20-0` a typo?

Reply:
The method I referred to is from here. https://github.com/cilium/cilium/blob/0e434b7c0a5f78b943006395997ddedab988d24b/install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml#L150